src/pages/profile/edit/profile-settings/index.tsx

```diff
@@ -1,16 +1,34 @@
 import Box from '@mui/material/Box';
 import FormControl from '@mui/material/FormControl';
 import MenuItem from '@mui/material/MenuItem';
-import Select from '@mui/material/Select';
+import Select, { SelectChangeEvent } from '@mui/material/Select';
 import Stack from '@mui/material/Stack';
 import Typography from '@mui/material/Typography';
+
+import { useCallback, useState } from 'react';
+
 import EditablePanel from '../EditablePanel';
+import useEditAccountMutation from '../../../../api/mutations/account/useEditAccountMutation';
 import { ProfileLang } from './ProfileLang';
 import { ProfileCurrency } from './ProfileCurrency';
-import { useCallback, useState } from 'react';
+import { useAppSelector } from '../../../../hooks/redux-hooks';
+import { getValueFromLocalStorage } from '../../../../utils';
+import { getProfile } from '../../../../stores/slices/authSlice';
+import { ThemeMode } from '../../../../types/profile.types';
+import { LOCAL_STORAGE_KEYS } from '../../../../config/local-storage.config';
 
 function ProfileSettings() {
-  const [theme, setTheme] = useState<'LIGHT' | 'DARK'>('LIGHT');
+  const profileId = useAppSelector(getProfile)?.id ?? '';
```

Comment:
I am not sure you need an empty string as a fallback here

It won't save you from the incorrect calling of mutation